mifosng-android/src/main/java/com/mifos/mifosxdroid/online/centerlist/CenterListFragment.java

```diff
@@ -203,6 +205,13 @@ public void reloadOnError() {
     @Override
     public void showCenters(List<Center> centers) {
         this.centers = centers;
+        Collections.sort(centers, new Comparator<Center>() {
+            @Override
+            public int compare(Center center1, Center center2) {
+                return center1.getName().toLowerCase().
+                        compareTo(center2.getName().toLowerCase());
+            }
```

Comment:
I didn't see overall codestyle, may be it is ok, but I would call input vars as first and second. Otherwise PR passed review